### PR TITLE
Integration test plan

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Might be easier to review it using this link:

https://github.com/vmware/vic/blob/feature/integration-test-documentation/tests/test-cases/Group1-Docker-Commands/TestCases.md

### CODE REVIEW

File: tests/test-cases/Group1-Docker-Commands/TestCases.md [hunk 1]
```diff
@@ -29,4 +29,22 @@ Group 1 - Docker Commands
 [Test 1-13 - Docker Version](1-13-Docker-Version.md)
 -
 [Test 1-14 - Docker Kill](1-14-Docker-Kill.md)
+-
```
Comment (jzt) [inline tests/test-cases/Group1-Docker-Commands/TestCases.md]: Minor: would it be possible to shrink the heading size for this list? The title is fine, but each list item seems jarringly large if you're used to looking at markdown on github. :)

**Comment by andrewtchin** [comment]
lgtm

**Comment by matthewavery** [comment]
LGTM this looks great! I know several of our calls are going to fail this initally due to error message format. Which is going to be good for cleaning up our output.

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**